packages/core-mobile/app/screens/send/utils/pvm/validate.ts

```diff
@@ -1,22 +1,24 @@
 import { Avalanche } from '@avalabs/core-wallets-sdk'
 import { TokenWithBalancePVM } from '@avalabs/vm-module-types'
-import { GAS_LIMIT_FOR_XP_CHAIN } from 'consts/fees'
 import { SendErrorMessage } from '../types'
 
 export const validate = ({
   amount,
   address,
   maxFee,
-  token
+  token,
+  estimatedFee,
+  gasPrice
 }: {
   amount: bigint
   address: string | undefined
   maxFee: bigint
   token: TokenWithBalancePVM
+  estimatedFee?: bigint
+  gasPrice?: bigint
 }): void => {
   if (!address) throw new Error(SendErrorMessage.ADDRESS_REQUIRED)
-
-  const fee = maxFee ? BigInt(GAS_LIMIT_FOR_XP_CHAIN) * maxFee : 0n
+  const fee = estimatedFee ?? maxFee
```

Comment:
I don't think these 2 are the same. maxFee is just the rate while estimated is the total fee, right?

Reply:
<img width="281" alt="Screenshot 2024-11-22 at 8 39 34 AM" src="https://github.com/user-attachments/assets/1c58584d-5f24-4919-b725-ddf6fbe7fea8">

maxFee is the defaultFeePerGas return from get-network-fee in vm-modules, which is always 1 pre-etna.  and estimatedFee is only calculated if the provider is etna provider, so it is really saying not etna (no estimatedFee), fallback to maxFee (static value)